_includes/head.html

```diff
@@ -19,6 +19,7 @@
 
 <link rel="stylesheet" href="{{ "css/orbit.css" }}">
 <link rel="stylesheet" href="{{ "css/theme-weebly.css" }}">
+<link rel="stylesheet" href="{{ "css/orbit-external.css" }}">
```

Comment:
Can you move `orbit-external` above `theme-weebly`. Theme Weebly will have overrides.